### PR TITLE
Install page improvements

### PR DESCRIPTION
**Pull request by yevgenykhazan**

### CODE REVIEW

File: docs/install/remote.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ You can write your code on a local machine while running your functions on a rem
 Before you begin, ensure that the following prerequisites are met:
 
 1. Applications:
-   - Python 3.7   
+   - Python 3.7 or later
```
Comment (gilad-shaham) [inline docs/install/remote.md:20]: We don't support any python release, because this is dev, we can change it to 3.9

Comment (yevgenykhazan) [reply docs/install/remote.md:20]: Fixed

File: docs/install/remote.md [hunk 2]
```diff
@@ -7,24 +7,24 @@ You can write your code on a local machine while running your functions on a rem
...
+   - [Using your IDE (e.g PyCharm or VSCode)](#using-your-ide-e-g-pycharm-or-vscode)
 
 <a id="prerequisites"></a>
 ## Prerequisite: set up your client
 
 Before you begin, ensure that the following prerequisites are met:
 
 1. Applications:
-   - Python 3.7   
+   - Python 3.9
```
Comment (AlonMaor14) [inline docs/install/remote.md:20]: python 3.9 is supported from mlrun 1.3.0 
otherwise, it should be recommended to use python 3.7

**Review by AlonMaor14 — COMMENTED** [review]
LGTM minor comment

**Review by AlonMaor14 — APPROVED** [review]

**Pull request merged**